rest/route/user.go

```diff
@@ -210,6 +210,29 @@ func (h *userPermissionsPostHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.NewTextInternalErrorResponse(err.Error())
 	}
 
+	// This is unfortunately very special-casey, but if the user has been granted permission to view/edit
+	// project settings, they also need to be given view access for the repo project, if applicable.
```

Comment:
By `if applicable` do you mean `if the project is attached`? Can we update this comment to be more specific about when someone is given view access?

Reply:
sure